doc/guides/backport-to-lts-branches.md

```diff
@@ -0,0 +1,124 @@
+# How to backport to LTS branches
+
+### Set up your repo
+
+- Collaborator access
+
+- Make sure nobody else is currently backporting to the same LTS branch
+
+- Clone node checkout staging branch
+  ```bash
+  git clone git@github.com/nodejs/node --origin DANGER
+  ```
+
+- Make sure up to date with staging
+  ```bash
+  git fetch --all
+  git checkout -B v8.x-staging DANGER/v8.x-staging
+  ```
+
+- Run the tests to make sure everything works:
+  ```bash
+  ./configure && make -j4 test
+  ```
+
+- Install branch-diff (and set up the key)
+  ```bash
+  npm install -g branch-diff
+  ```
+
+### Triage PRs
+
+Go through [PRs with the v8.x label](https://github.com/nodejs/node/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Av8.x), for each PR:
+
+- Run CI
+- Check approvals (you can approve yourself)
+- Check SemVer
+- Check the PR metadata wasn't changed from `master` commit.
+- If there are merge conflicts, comment in the issue asking the PR author to rebase.
+- Land if okay
+  - Add git alias if you haven't already:
+  ```bash
+  git config --global alias.pa '!curl -L $1.patch | git am -3 -S --whitespace=fix #'
```

Comment:
I've ended up removing `--whitespace=fix` as it can cause some serious issues with certain backports.